test/parser.test.js

```diff
@@ -2530,7 +2530,23 @@ describe('parser', () => {
       i18nextParser.end(fakeFile)
     })
 
-    it('emits a `warning` event if a namespace contains a variable', (done) => {
+    it('emits a `warning` event if ns prop contains a variable', (done) => {
```

Comment:
I might be missing something but it looks to me that this test is exactly the same as the following one, isn't it?

Reply:
if there's a change requested for this i'll gladly make it, we'd love to have this so we could get back on the main branch of this library =)

Reply:
I must have been tired when I reviewed it. Will merge it now